polls/tests/test_voting.py

```diff
@@ -75,7 +75,7 @@ def test_voting_once_only(self):
         self.assertContains(response, 'is this a test')
         response = client.post(reverse('molo.polls:vote',
                                kwargs={'question_id': question.id}))
-        self.assertContains(response, "You didn&#39;t select a choice")
+        self.assertContains(response, "field is required")
```

Comment:
A bit bikesheddy, but `You didn't select a choice` seems a bit more descriptive an error than `... field is required` for users to see. Should we maybe keep that error message the same?